src/components/form/configUIForm.tsx

```diff
@@ -38,6 +39,7 @@ const FormStepper = React.lazy(() =>
  */
 export interface ConfigUIFormProps {
   prepareSubmitValues: (
+    config: PluginConfig,
```

Comment:
confirming this is the breaking change - yes?

Reply:
yes, I just don't see any other way to get config, so people must explicitly pass it in their `prepareSubmitValues` function from the arguments to `corePrepareSubmitValues`

Reply:
agreed on that 👍